it/php/src/main.php

```diff
@@ -0,0 +1,16 @@
+<?php
+    namespace Integration\Test;
+    use Integration\Test\Client\ApiClient;
+    use Microsoft\Kiota\Authentication\Oauth\ClientCredentialContext;
+    use Microsoft\Kiota\Authentication\PhpLeagueAuthenticationProvider;
+    use Microsoft\Kiota\Http\GuzzleRequestAdapter;
+    $tokenRequestContext = new ClientCredentialContext(
+        'tenantId',
+        'clientId',
+        'clientSecret'
+    );
+    $authProvider = new PhpLeagueAuthenticationProvider($tokenRequestContext, [], []);
+    $requestAdapter = new GuzzleRequestAdapter($authProvider);
+    $client = new ApiClient($requestAdapter);
+    echo "Hello {$client}";
```

Comment:
Bug?

Reply:
it's so I don't get a linting error of unused variable